### ISSUE TITLE
user_keys() in salt.modules.ssh missing strip()

### ISSUE THREAD

**Issue by kevinquinnyo** [saltstack/salt#32627]
### Description of Issue/Question
When retreiving keys from the `user_keys()` module, there is a trailing newline.  This should be stripped.
### Steps to Reproduce Issue
Easy to reproduce:
```python
#!py

def run():
    states = {}
    ssh_keys = __salt__['ssh.user_keys']('root', 'id_rsa.pub')
    public_key = ssh_keys['root']['id_rsa.pub']
    states['test print ssh key'] = {
        'cmd.run': [
            {'name': 'echo "{0}"'.format(public_key)}
        ]
    }
    return states
```
Notice the trailing newline.  This can be fixed with a simple `.strip()` here:
```
keys[u][keyname] = ''.join(_fh.readlines()).strip()
```

### Versions Report
This appears to be in all versions.

**Comment by kevinquinnyo** [comment]
I doubt any tests are needed for this one, since anyone else who may have discovered this in the past is just adding a `strip()` themselves, and of course that isn't a problem:

```python
print 'asdf\n'.strip().strip()
```
So no chance of regression for this simple fix.

**Comment by cachedout** [comment]
Fixed via #32628 I believe. Closing.
